receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -406,17 +410,217 @@ func TestTranslateV2(t *testing.T) {
 				return expected
 			}(),
 		},
+		{
+			name: "service with target_info metric",
+			request: &writev2.Request{
+				Symbols: []string{
+					"",
+					"job", "production/service_a", // 1, 2
+					"instance", "host1", // 3, 4
+					"machine_type", "n1-standard-1", // 5, 6
+					"cloud_provider", "gcp", // 7, 8
+					"region", "us-central1", // 9, 10
+					"datacenter", "sdc", // 11, 12
+					"__name__", "normal_metric", // 13, 14
+					"d", "e", // 15, 16
+					"__name__", "target_info", // 17, 18
+				},
+				Timeseries: []writev2.TimeSeries{
+					// Generating 2 metrics, one have the target_info in the name and the other is a normal gauge.
+					// The target_info metric should be translated to use the resource attributes.
+					// The normal_metric should be translated as usual.
+					{
+						// target_info metric
+						Metadata:   writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_INFO},
```

Comment:
`writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_INFO}` I'm setting the type as METRIC_TYPE_INFO, but, when a metric comes as a `target_info` which type it carries? Is it matter ?

Reply:
It is a gauge, following the current spec: https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#info

Let's use Gauge while Prometheus hasn't implemented the `Info` type

Reply:
Actually the type doesn't feel important at the moment, we're only checking the metric name 🤔

Reply:
[3ce627e](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38812/commits/3ce627e2a4ec12d841341eda1dc07deb8cab4f7e)